content/docs/tutorials/get-started/data-access.md

```diff
@@ -40,16 +48,14 @@ When working inside another DVC project though, this is not the best strategy
 because the connection between the projects is lost — others won't know where
 the data came from or whether new versions are available.
 
-## Import the dataset
-
-> Requires an [initialized](/doc/tutorials/get-started#initialize) <abbr>DVC
-> project</abbr>.
+## Import file or directory
 
-`dvc import` downloads a dataset, while also tracking it **in the same step**:
+`dvc import` downloads any file or directory similar to `dvc get`, while also
+creates a `.dvc` file that can be saved in the project:
```

Comment:
```suggestion
`dvc import` also downloads any file or directory, while also
creating a `.dvc` file that can be saved in the project:
```
Removed the `dvc get` part here because it's mentioned in the next paragraph (after the code block).